commons-test/pom.xml

```diff
@@ -25,8 +25,13 @@
          <scope>provided</scope>
       </dependency>
       <dependency>
-         <groupId>junit</groupId>
-         <artifactId>junit</artifactId>
+         <groupId>org.junit.jupiter</groupId>
+         <artifactId>junit-jupiter-engine</artifactId>
+         <scope>provided</scope>
+      </dependency>
+      <dependency>
+         <groupId>org.junit.vintage</groupId>
+         <artifactId>junit-vintage-engine</artifactId>
```

Comment:
I don't think adding the dependencies here is enough, you also have to add them to the surefire plugin instead of the `surefire-junit47` dependency:

https://junit.org/junit5/docs/current/user-guide/#running-tests-build-maven

Reply:
#8177 the JUnit 5 component upgrade is in another PR now